src/components/manage/Contents/Contents.jsx

```diff
@@ -1150,16 +1162,17 @@ class Contents extends Component {
                                 'EffectiveDate',
                                 'CreationDate',
                                 'ModificationDate',
-                                'Type',
+                                'portal_type',
                               ],
                               index => (
-                                <Dropdown.Item key={index}>
+                                <Dropdown.Item key={index} data-cy={index}>
                                   <Icon name="dropdown" />
                                   <FormattedMessage id={Indexes[index].label} />
                                   <Dropdown.Menu>
                                     <Dropdown.Item
                                       onClick={this.onSortItems}
-                                      value={`${index}|ascending`}
+                                      value={`${Indexes[index].sort_on}|ascending`}
+                                      data-cy={`${Indexes[index].sort_on}|ascending`}
```

Comment:
@ksuess can't we just rely on "value" here instead of "data-cy"? I mean we already have a way of getting this exact Dropdown.Item, why add a second one? I know the folks at Cypress recommend this. Though, this is the exact reason why I don't like their recommendation. It makes people think that they should always rely on data-cy attributes (which is a bad thing IMHO), not use them as a last resort.

Reply:
index iterates 
                              [
                                'id',
                                'sortable_title',
                                'EffectiveDate',
                                'CreationDate',
                                'ModificationDate',
                                'portal_type',
                              ]